content/influxdb/v2.0/reference/flux/stdlib/testing/benchmark.md

```diff
@@ -0,0 +1,69 @@
+---
+title: testing.benchmark() function
+description: >
+  The `testing.benchmark()` function allows for accurate benchmark tests by executing
+  a test case without the additional overhead of comparing input and output tables.
+menu:
+  influxdb_2_0_ref:
+    name: testing.benchmark
+    parent: Testing
+weight: 301
+---
+
+The `testing.benchmark()` function allows for accurate benchmark tests by executing
+a test case without the additional overhead of comparing input and output tables.
+
+_**Function type:** Test_  
+
+```js
+import "testing"
+
+testing.benchmark(
+  case: exampleTestCase
+)
+```
+
+## Parameters
+
+### case
+Test case to benchmark.
+
+_**Data type:** Function_
+
+## Examples
+
+##### Define and benchmark a test case
```

Comment:
Thoughts on adding a bit more description upfront about the test case here? Perhaps what data is input/ouput, what's being compared, and results returned